Main.hs

```diff
@@ -8,6 +8,7 @@ import Distribution.Simple.PackageDescription
 import Distribution.PackageDescription
 import Distribution.PackageDescription.Parsec
 import Distribution.PackageDescription.PrettyPrint
+import Distribution.Types.ExeDependency
```

Comment:
Wait this belongs in the other PR, right?

Reply:
There's no other PR, this is required to make the new CI pass in any case. Didn't catch this in time for #20, so it's a separate commit unfortunately.